docs/SETUP.md

```diff
@@ -70,6 +70,25 @@ We use Postgres around these parts. Installation will differ based on your OS.
   * Other [linux instructions](https://www.postgresql.org/download/) (server and developer libraries needed)
 * Run the command `rails db:create && rails db:migrate` to set up the tables
 
+* Configure Postgres
+ * Setup your user; the suggested password is `postgres` if you only use this machine for development purposes
+   ```
+   sudo -u postgres createuser -s `whoami`
+   ```
+ * Set your env variables:
+   * Mac:
+     ```
+     echo "export POSTGRES_USER=`whoami`" >>~/.bashrc
+     echo "export POSTGRES_PASSWORD=postgres" >>~/.bashrc
+     echo "export PGDATABASE=dcaf_case_management_development" >>~/.bashrc
+     ```
+   * Ubuntu:
+     ```
+     echo "export POSTGRES_USER=`whoami`" >>~/.bashrc
+     echo "export POSTGRES_PASSWORD=postgres" >>~/.bashrc
+     echo "export PGDATABASE=dcaf_case_management_development" >>~/.bashrc
```

Comment:
This one makes your `psql` command work without having to pass `-d dcaf_case_management_development`

Reply:
`rails db` also works for this fwiw.

This doesn't interact with the test env, right? The `database.yml` overrules it?